### PR TITLE
Feature: format with prettier

### PR DESCRIPTION
**Pull request by andrewqiamzn**

### CODE REVIEW

**Review by giotar — CHANGES_REQUESTED** [review]
The prettified code looks good, but you need to add the `.prettierrc` file, add the prettier dependency in `package.json` and possibly also a couple of NPM scripts to check and fix the format

File: adapters/dab-adb-bridge/src/package.json [hunk 1]
```diff
@@ -36,6 +38,9 @@
   "devDependencies": {
     "@types/config": "^3.3.0",
     "@types/node": "^16.11.56",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-prettier": "^3.4.0",
```
Comment (giotar) [inline adapters/dab-adb-bridge/src/package.json:42]: I don't think you need these two ESLint plugins, do you?

**Review by giotar — APPROVED** [review]

File: adapters/dab-adb-bridge/src/package.json (hunk 1 above)
Comment (andrewqiamzn) [reply adapters/dab-adb-bridge/src/package.json:42]: good spot! I copied and pasted from our ConciergeClient package.json, but it seems like we don't need the `eslint-plugin-prettier`

Comment (andrewqiamzn) [reply adapters/dab-adb-bridge/src/package.json]: fixed!

**Pull request merged**